docs/content/guides/dagster/example_repo.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Example Dagster Repo | Dagster
+description: This guide describes our example Dagster repository
+---
+
+# Hacker News Example Repository
+
+The [Hacker News example repo](https://github.com/dagster-io/dagster/tree/master/examples/hacker_news) takes advantage of a wide range of Dagster features, and can be useful as a point of reference for how we imagine structuring Dagster projects and using different APIs. At a high level, it consists of three pipelines, all centered around a (contrived) use case of downloading Hacker News user activity data, loading it into a database, and doing analysis on the aggregated data.
```

Comment:
Nitpick: I think no comma needed after "features".

Reply:
I would mention "training an ML model" or "building a recommender model" or something - I don't think that necessarily fits under analysis.